client/components/Layout/VolumeSlider.vue

```diff
@@ -60,13 +60,4 @@ export default Vue.extend({
 .volume-slider {
   width: 10em;
 }
-
-// HACK: https://github.com/vuetifyjs/vuetify/issues/8436.
-// https://vuetifyjs.com/en/api/v-btn/#retain-focus-on-click prop was added
-// but it seems we're using a prop combination that it's incompatible with it.
-
-// SO link: https://stackoverflow.com/questions/57830767/is-it-default-for-vuetify-to-keep-active-state-on-buttons-after-click-how-do-yo/57831256#57831256
-.active-button:focus::before {
```

Comment:
Did you check removing this actually works? I'm not sure if this was left here because the styling had to be scoped.

Reply:
It worked for me, is it not working on the demo for you?

Reply:
I confirm it is

Reply:
It is or is not working?

Reply:
It is working :)

I'll approve as my complaints are now adressed, forgot to do that :)